_data/sites.json

```diff
@@ -14916,6 +14916,18 @@
         ]
     },
 
+    {
+        "name": "Wakanim",
+        "url": "https://www.wakanim.tv/sc/v2/static/contactus",
+        "url_fr": "https://www.wakanim.tv/fr/v2/static/contactus",
+        "url_de": "https://www.wakanim.tv/de/v2/static/contactus",
+        "difficulty": "medium",
+        "notes": "Fill the form with your username and mail, select category \"Account/Login\" and subcategory \"Account Deletion\". An e-mail confirmation will be sent.",
```

Comment:
Is the e-mail confirmation just a note saying the account was deleted or something to take action to confirm the request?

Reply:
Oh, great! Thanks so much for your help. :-) I've tried to make the notes a bit clearer. It's basically like opening a ticket. First it's just a note that they'll take care of it. Then there's another note that the request is completed.

Reply:
Ok, so then I think we should change it to hard, as it requires human intervention on the other side.